Chatkit.xcodeproj/project.pbxproj

```diff
@@ -722,6 +695,13 @@
 			path = Chatkit;
 			sourceTree = "<group>";
 		};
+		22B2D2AE23E2FBF10085D79C /* Frameworks */ = {
+			isa = PBXGroup;
+			children = (
+			);
+			name = Frameworks;
+			sourceTree = "<group>";
+		};
```

Comment:
Really not a big deal but we this added `Frameworks` group is redundant.

Reply:
This has been added automatically by Xcode. I will remove this group.